inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/network/Sender.java

```diff
@@ -726,4 +736,26 @@ public boolean isIdleClient(NettyClient client) {
         return true;
     }
 
+    private NettyClient getClient(LoadBalance loadBalance, EncodeObject encodeObject) {
```

Comment:
Is it better to remove this into `ClientMgr`? Then,  methods such as `asyncSendMessage` can directly use previous signature, and no need to add additional parameter.